styles/modules/modals/_about.scss

```diff
@@ -0,0 +1,42 @@
+.about { 
+  .brandSmall {
+    width: 50px;
+    height: 50px;
+  }
+
+  .gemSmall {
+    width: 16px;
+    height: 16px;
+    margin: 1px;
+    background-size: contain;
+  }
+
+  #donation-qr-code {
+    width: 200px; 
+    height: 200px;
+    background: black;
+  } 
+
+  .contributorList li {
+    padding-left: $padLg;
+    padding-bottom: $padSm;
+  }
+
+  #donationAddress {
+    color: $text2;
+  }
+
+  #copyNotification {
+    @extend .clrBr, .border;
```

Comment:
Same thing with the clrBr class. It should be put on the span.

Basically, any of the classes in _theme.scss need to be applied directly in the markup. That way when we override those classes when a new theme is applied, the elements will take on the overridden color.

Doing it in the style sheet like this means that, at compile time, SASS will just compile out the default clrBr color and embed it in the #copyNotification selector and it would ignore the fact that we may override that class at some point at run-time.